include/robot_interfaces/status.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
```

Comment:
The file should probably have a license header like the other ones.

Reply:
indeed